Odata-docs/client/v7/batch-operations.md

```diff
@@ -14,43 +14,44 @@ ms.topic: article
 OData Client for .NET supports batch processing of requests to an OData service. This ensures that all operations in the batch are sent to the data service in a single HTTP request, enables the server to process the operations atomically, and reduces the number of round trips to the service.
 
 OData Client for .NET doesn't support sending both query and change in one batch request.
+
+There are 2 ways of invoking batch operations in odata client:
+1. Batch Query
+2. Batch Modification
  
 ## Batch Query 
 
 To execute multiple queries in a single batch, you must create each query in the batch as a separate instance of the `DataServiceRequest<TElement>` class. The batched query requests are sent to the data service when the `ExecuteBatch` method is called. It contains the query request objects. 
 
 This method accepts an array of `DataServiceRequest` as parameters. It returns a `DataServiceResponse` object, which is a collection of `QueryOperationResponse<T>` objects that represent responses to individual queries in the batch, each of which contains either a collection of objects returned by the query or error information. When any single query operation in the batch fails, error information is returned in the `QueryOperationResponse<T>` object for the operation that failed and the remaining operations are still executed. 
 
-```c#
-
-    DefaultContainer dsc = new DefaultContainer(new Uri("https://services.odata.org/V4/(S(uvf1y321yx031rnxmcbqmlxw))/TripPinServiceRW/"));
-    public void BatchQuery()
-    {
-        var peopleQuery = dsc.People;
-        var airlinesQuery = dsc.Airlines;
-
-        var batchResponse = dsc.ExecuteBatch(peopleQuery, airlinesQuery);
-        foreach(var r in batchResponse)
-        {
-            var people = r as QueryOperationResponse<Person>;
-            if (people != null)
-            {
-                foreach (Person p in people)
-                {
-                    Console.WriteLine(p.UserName);
-                }
-            }
-
-            var airlines = r as QueryOperationResponse<Airline>;
-            if (airlines != null)
-            {
-                foreach (var airline in airlines)
-                {
-                    Console.WriteLine(airline.Name);
-                }
-            }
-        }
-    }
+``` csharp
+DefaultContainer dsc = new DefaultContainer(new Uri("https://services.odata.org/V4/(S(uvf1y321yx031rnxmcbqmlxw))/TripPinServiceRW/"));
+
+DataServiceQuery<Person> peopleQuery = dsc.People;
+DataServiceQuery<Airline> airlinesQuery = dsc.Airlines;
+
+DataServiceResponse batchResponse = dsc.ExecuteBatch(peopleQuery, airlinesQuery);
```

Comment:
With .netcore it might be worth considering using the await/async syntax? For example since only `dsc.ExecuteBatchAync` is available in .netcore

Reply:
I have added a note.

Reply:
#Resolved